src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -184,35 +182,35 @@ export default class TacticSelectorContent extends Vue {
         break
     }
   }
-  tacticMenuColour (tactic: Tactic, item: TacticMenuItem) {
+  tacticMenuColour (tactic: Tactic, item: MenuItem) {
     if (item.action === TacticMenuOptions.PIN && tactic.pinned) {
```

Comment:
What is this `PIN`.

Reply:
Pinning Tactic to our short cut bar at the bottom of the application.